test/full-tracer/tx-hash.test.js

```diff
@@ -0,0 +1,19 @@
+const chai = require("chai");
+const expect = chai.expect;
+const { Scalar } = require("ffjavascript");
+const { ethers } = require("ethers");
+const txs = require("./tx-hash.json")
+const { getTransactionHash } = require("../../src/sm/sm_main/debug/full-tracer-utils")
+describe("tx hash tests", async function () {
+    this.timeout(10000000);
+
+    it("Should test al tx hash", async () => {
+
+        for (let test of txs) {
+            const txHash = getTransactionHash(test.to, Number(test.value), Number(test.nonce), test.gasLimit, test.gasPrice, test.data, test.r, test.s, test.v)
```

Comment:
Could we change `value` and `nonce` in the JSON to be numbers ?
Just like `chainId` in the first element in the JSON.
Also, `chainId` is in two formats in the JSON, number (`5`) and string (`0x05`)